common/src/commonMain/kotlin/com/trikot/sample/repositories/impl/QuoteRepositoryImpl.kt

```diff
@@ -12,8 +12,7 @@ class QuoteRepositoryImpl() : QuoteRepository {
     override fun getQuotes(): Publisher<List<Quote>> {
         return ColdPublisher { cancellableManager ->
             val request = RequestBuilder().also {
-                it.baseUrl = "https://breaking-bad-quotes.herokuapp.com"
-                it.path = "/v1/quotes/5"
+                it.baseUrl = "https://breaking-bad-quotes.herokuapp.com/v1/quotes/5"
```

Comment:
I will have keep the path separated from the baseUrl 🤷‍♂️

Reply:
That's actually a bug in a specific version of trikot http, you can keep them separate if you use the latest version